auslib/admin/views/forms.py

```diff
@@ -155,6 +158,33 @@ class ExistingPermissionForm(DbEditableForm):
     options = JSONStringField(None, 'Options')
 
 
+class ScheduledChangeNewPermissionForm(ScheduledChangeTimeForm, NewPermissionForm):
+    permission = StringField('Permission', validators=[Length(0, 50), InputRequired()])
+    username = StringField('Username', validators=[Length(0, 100), InputRequired()])
+    change_type = SelectField("Change Type", choices=[('insert', 'insert'), ('update', 'update'), ('delete'), ('delete')])
+
+
+class ScheduledChangeExistingPermissionForm(ScheduledChangeTimeForm, ExistingPermissionForm):
+    permission = StringField('Permission', validators=[Length(0, 50), InputRequired()])
+    username = StringField('Username', validators=[Length(0, 100), InputRequired()])
+    change_type = SelectField("Change Type", choices=[('insert', 'insert'), ('update', 'update'), ('delete'), ('delete')])
+
+
+class ScheduledChangeDeletePermissionForm(ScheduledChangeTimeForm):
+    change_type = SelectField("Change Type", choices=[('insert', 'insert'), ('update', 'update'), ('delete', 'delete')])
+    permission = StringField('Permission', validators=[Length(0, 50), InputRequired()])
+    username = StringField('Username', validators=[Length(0, 100), InputRequired()])
+    data_version = IntegerField('data_version', validators=[InputRequired()], widget=HiddenInput())
+
+
+class EditScheduledChangeNewPermissionForm(ScheduledChangeTimeForm, NewPermissionForm):
+    sc_data_version = IntegerField('sc_data_version', validators=[InputRequired()], widget=HiddenInput())
```

Comment:
Yeah, I see what you mean. The difference here is that the existing Permissions forms didn't have anything that wasn't desired is the Scheduled Changes versions, whereas the Releases ones did (eg: alias, "blob" instead of "data"). Perhaps it's less confusing to explicitly specify them rather than inheriting, though.